File: Clases 2018/Clases-C#/ClasePractica12_2018 (transacciones con data manager)/C#_TPS_BugTracker/TPS_BugTracker/BusinessLayer/Entities/HistorialBug.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TPS_BugTracker.BusinessLayer.Entities
{
    public class HistorialBug
    {
        public int id_detalle { get; set; }
        public int responsable { get; set; }
        // Nombre del usuario responsable de generar el historial del bug
        public string n_responsable { get; set; }
        public DateTime fecha { get; set; }
        public int asignado_a { get; set; }
        // Nombre del usuario asignado a resolver el bug en el historial
        public string n_asignado_a { get; set; }
        public int estado { get; set; }
        // Nombre del estado
        public string n_estado { get; set; }
    }

}

File: PAV 1-TPI - CASOS DE USO version 1/Resueltos/Usuarios+SeleccionActBugs/App_BugTracker resuelto/App_BugTracker/BusinessLayer/Entites/HistorialBug.vb
Public Class HistorialBug
    Public Property id_detalle As Integer
    Public Property responsable As Integer
    'Nombre del usuario responsable de generar el historial del bug
    Public Property n_responsable As String
    Public Property fecha As Date
    Public Property asignado_a As Integer
    'Nombre del usuario asignado a resolver el bug en el historial
    Public Property n_asignado_a As String
    Public Property estado As Integer
    'Nombre del estado
    Public Property n_estado As String
End Class
